shell/client/grain/grainlist-client.js

```diff
@@ -470,9 +551,65 @@ Template.sandstormGrainTable.helpers({
     // TODO(cleanup): access Meteor.user() through db object
     return this.size && prettySize(this.size);
   },
+
+  sortedGrains(grains) {
+    const instance = Template.instance();
+    return sortGrains(grains, [instance.sortOrder.get()]);
+  },
+
+  sortOrder() {
+    const instance = Template.instance();
+    return instance.sortOrder.get();
+  },
+
+  equal(a, b) {
+    return a === b;
+  },
 });
 
 Template.sandstormGrainTable.events({
+  "click thead td.grain-name"(evt) {
```

Comment:
Should we be wrapping each heading in `<button>` for accessibility and mobile touch target hinting?